### PR TITLE
[Author] Update hls.js#package.json

### PR DESCRIPTION
**Pull request by dighan**
# Pull request template for new library

- Point to `video-dev` rather than `dailymotion` org
- Align `description` and `license` field with `video-dev/hls.js#package.json`

Pull request for issue: https://github.com/cdnjs/cdnjs/issues/12927

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/video-dev/hls.js
* Official website (optional, not the repository): https://video-dev.github.io/hls.js/stable/demo/
* NPM package url (optional): https://www.npmjs.com/package/hls.js
* License and its reference: `Apache-2.0`
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 323
  * Count of stars: 4866
  * Count of forks: 1118
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week:
  * Downloads in the last month:

## Essential checklist

* [x] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [ ] This lib was not found on cdnjs repo
* [ ] Doesn't already exist / has duplicate issue and PR
* [ ] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/hls.js/package.json [hunk 1]
```diff
@@ -2,11 +2,11 @@
   "name": "hls.js",
   "version": "0.10.1",
```
Comment (dighan) [inline ajax/libs/hls.js/package.json]: Should I update the `version` even if `autoupdate` is specified ?

Comment (PeterDaveHello) [reply ajax/libs/hls.js/package.json]: no need

**Comment by PeterDaveHello** [comment]
The auto-update config matches nothing here: https://github.com/video-dev/hls.js/tree/v0.11.0, so no new version will be automatically added. The info should be updated already.

**Comment by PeterDaveHello** [comment]
@dighan the tag exists but the files seem not match the auto-update config map

**Comment by dighan** [comment]
You're right @PeterDaveHello. I discuss with the team to find out the proper solution. Thanks again for your support 👍

**Comment by sashberd** [comment]
@dighan As I said in our issue conversation it is better to use npm update. But you are ruler 😄

**Pull request merged**